### ISSUE TITLE
Playground: No Errors, produces JS

### ISSUE THREAD

**Issue by sergeykish** [microsoft/TypeScript-Website#580]
When I input `1 / "foo"`
And click Run
It shows Error message
And no compiled JS

https://www.typescriptlang.org/play/
![image](https://user-images.githubusercontent.com/17841/81914874-88dfd100-95da-11ea-98b9-2205f37cca36.png)

https://www.typescriptlang.org/v2/en/play JS:
![image](https://user-images.githubusercontent.com/17841/81915343-1de2ca00-95db-11ea-9332-3ea1bbb115a9.png)

https://www.typescriptlang.org/v2/en/play Errors:
![image](https://user-images.githubusercontent.com/17841/81914997-ac0a8080-95da-11ea-8b12-90e7c13b5eae.png)

**Repro:**

Firefox 77
Opera 68

**Code**

```
> 1 / "foo"
index.ts:3:5 - error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
```

**Expected behavior:**
it shows Errors, empty JS

**Actual behavior:**
v1: it produces JS, no way to check Errors
v2: it produces JS, I can check Errors, though they are not shown by default

**Comment by orta** [comment]
I think this is as expected: TypeScript will _always_ produce JavaScript for code (assuming the code has correct syntax) even if the compiler gives errors. This is a goal of TypeScript to have JavaScript compatibility.

**Comment by sergeykish** [comment]
Maybe your workflow differs from mine.

    $ tsc foo.ts 
    foo.ts:1:5 - error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
    
    1 1 / "foo"
          ~~~~~
    
    
    Found 1 error.

Yes, it created `foo.js` but there is big error message. Hard to miss. It is different in playground, I've honestly thought my code (not this one) had no errors.

And I'm not alone [tsc as linter](https://stackoverflow.com/questions/55002137/typescript-noemit-use-case). There are `--noEmit` and `--noEmitOnError`.

Ok, not my workflow, I'll point people to other tools - [typescript playground](https://tio.run/##K6ksSC1OLsosKPn/31BBX0EpLT9f6f//1JzU3NS8EuXMFL3knMTiYkMIpWsUnVhSUpSZVFqSahgbDVRrq5SUWKQUCwA), [flow playground](https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBGYYBEU5w5A).

**Comment by orta** [comment]
I'm not sure I get what you mean? There are red underlines on every screenshot, and the JS is generated just like how the CLI works - the playground is meant to be consistent with that

**Comment by sergeykish** [comment]
I did not notice red underline, maybe because they look like spellcheck errors in browser context. I remember red box on the scrollbar, clicked it with no result, searched menus. I honestly thought your application is broken, checked web console

    Error: Can only have one anonymous define call per script file
    SyntaxError: "JSON.parse: unexpected end of data at line 1 column 1 of the JSON data" main-3.js:523:17
    Error: Could not find source file: 'file:///output.js'.
    (Firefox 76)

Finally I've found "Try Playground v3 beta" link, it was better, still gives feedback not as quick as CLI REPL. My use case was about showing what TS forbids (online) to novices. In playground v3 it requires constant check of Errors tabs, impossible in v2.

Maybe that's rare use case but that's my experience, hope it help future visitors.
